Sources/ArcGISToolkit/Components/Offline/MetadataDetailView.swift

```diff
@@ -0,0 +1,128 @@
+    @ObservedObject var model: PreplannedMapModel
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss: DismissAction
+    
+    public var body: some View {
+        Form {
+            Section {
+                VStack(alignment: .leading) {
+                    if let image = model.preplannedMapArea.thumbnail {
+                        VStack(alignment: .leading, spacing: 0) {
+                            Text("Thumbnail")
+                                .font(.caption)
+                                .foregroundStyle(.secondary)
+                            LoadableImageView(loadableImage: image)
+                                .clipShape(.rect(cornerRadius: 10))
+                                .padding(.vertical, 10)
+                        }
+                        Divider()
+                    }
+                    VStack(alignment: .leading, spacing: 0) {
+                        Text("Name")
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                        Text(model.preplannedMapArea.title)
+                            .font(.subheadline)
+                    }
+                    Divider()
+                    VStack(alignment: .leading, spacing: 0) {
+                        Text("Description")
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                        Text(model.preplannedMapArea.description)
+                            .font(.subheadline)
+                    }
+                    Divider()
+                    if let size = model.directorySize {
+                        VStack(alignment: .leading, spacing: 0) {
+                            Text("Size")
+                                .font(.caption)
+                                .foregroundStyle(.secondary)
+                            Text(size.formatted(.byteCount(style: .file, allowedUnits: [.kb, .mb])))
+                                .font(.subheadline)
+                        }
+                    }
+                }
+            }
+            Section {
+                HStack {
+                    ZStack {
+                        Image(systemName: "circle.fill")
+                            .foregroundStyle(.secondary)
+                            .opacity(0.1)
+                            .font(.title)
+                        Image(systemName: "trash.fill")
+                            .foregroundStyle(.red)
+                            .imageScale(.medium)
+                    }
+                    Button {
+                        dismiss()
+                        Task {
+                            await model.removeDownloadedPreplannedMapArea()
+                        }
+                    } label: {
+                        Text("Delete Map Area")
+                            .font(.subheadline)
+                            .foregroundColor(.red)
```

Comment:
What does the difference in appearance look like between destructive and setting the color manually? I would lean towards using built-in paradigms where possible instead of hard-coding colors/sizes/styles/etc.

Reply:
You are right - destructive looks fine. I'll revert back to destructive.